workers/datasets_based/src/datasets_based/workers/first_rows.py

```diff
@@ -459,12 +459,12 @@ def compute_first_rows_response(
     try:
         split_full_names = get_dataset_split_full_names(dataset=dataset, use_auth_token=use_auth_token)
     except _EmptyDatasetError as err:
-        raise EmptyDatasetError("The dataset is empty.", cause=err) from err
+        raise EmptyDatasetError(f"The dataset '{dataset}' is empty.", cause=err) from err
     except Exception as err:
-        raise SplitsNamesError("Cannot get the split names for the dataset.", cause=err) from err
+        raise SplitsNamesError(f"Cannot get the split names for the dataset '{dataset}'.", cause=err) from err
     # ^ can raise DatasetNotFoundError or SplitsNamesError
     if config not in [split_full_name["config"] for split_full_name in split_full_names]:
-        raise ConfigNotFoundError(f"config {config} does not exist for dataset {dataset}")
+        raise ConfigNotFoundError(f"The config '{config}' does not exist for dataset '{dataset}'")
```

Comment:
I saw `{dataset}` was passed in the error message here, so I included it in other error messages. But I would like to be sure this is right.
- For a specific user, it does not make sense to tell them the name of their dataset
- But for our cache, it might be useful information
To be confirmed.

Reply:
Nope, it's not necessary since the cache entry in mongo already contains dataset/config/split: https://github.com/huggingface/datasets-server/blob/main/libs/libcommon/src/libcommon/simple_cache.py#L69

This means that giving the config is also redundant.

Reply:
@severo I agree with dataset. On the contrary, the config is redundant for our cache, but it might be informative for the user getting it in the error message, no?

Reply:
Oh, yes, you're right